.github/workflows/precommit.yaml

```diff
@@ -21,3 +21,7 @@ jobs:
         run: pip install -r requirements.txt
       - name: Run pre-commit
         run: pre-commit run --all-files
+      - name: Run pre-commit on the latest commit message
+        run:
+          pre-commit run gitlint --hook-stage commit-msg --commit-msg-filename
```

Comment:
Are we using the default gitlint rules here? Do they fit us?

Reply:
Currently using default ones, yes; they should fit our contribution guide.